docs/content/tutorials/tutorial-add-radius/index.md

```diff
@@ -0,0 +1,303 @@
+---
+type: docs
+title: "Tutorial: Add Radius to an existing application"
+linkTitle: "Existing application"
+description: "Learn how to incrementally add Radius to an existing application"
+weight: 600
+slug: "add-radius"
+categories: "Tutorial"
+tags : ["kubernetes","kubectl"]
+---
+
+This tutorial will teach you about adding Radius to an existing application using the built-in Kubernetes integration functionality. You will learn how to:
+
+1. Deploy an existing Kubernetes application
+1. Run and test the application
+1. Add Radius to the deployed application
+1. Confirm that Radius was added and can detect the application
+
+By the end of this tutorial, you will have deployed an existing Kubernetes application and then updated it to add Radius functionality without changing the application or its original Kubernetes configurations.
+
+## Prerequisites
+
+- [Kubernetes cluster]({{< ref "guides/operations/kubernetes/overview#supported-kubernetes-clusters" >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [rad CLI]({{< ref howto-rad-cli >}})
+
+## Overview
+
+As a part of this tutorial you will deploy an existing containerized Guestbook application originally authored by the Kubernetes community for use in their own tutorials. You will then add Radius to the deployed application. To incrementally add Radius to an existing application, you will leverage the built-in Kubernetes integration functionality by adding an annotation to the application's Kubernetes deployment manifest. This approach is particularly useful for adding Radius capabilities to applications that are already deployed.
+
+The Guestbook application consists of a web front end along with primary and secondary Redis containers for storage, all deployed with Kubernetes. For more information about the application and access its source code, see the [Kubernetes tutorial](https://kubernetes.io/docs/tutorials/stateless-application/guestbook/) and their [examples repo](https://github.com/kubernetes/examples/tree/master/guestbook).
+
```

Comment:
I would be good to have a diagram here if possible. We've consistently gotten feedback on tutorials that the diagram is needed.

Reply:
(non-blocking for this PR)